posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+---
+authors:
+- Anthony Rey
+tags:
+- Atomic Design
+- Front-End
+- Bootstrap
+date: 2020-01-16T15:09:00.000Z
+title: "The cohabitation between Atomic Design and Bootstrap"
+image: https://raw.githubusercontent.com/gnuk/blog-usa/atomic-bootstrap/images/2020/01/atomic-design-bootstrap.png
+---
+
+My previous [blog post](https://blog.ippon.tech/atomic-design-in-practice/) was about Atomic Design in practice. I used this organisational method for several years in many teams and different projects.
+
+At the same time, often, colleagues are promoting [Bootstrap](https://getbootstrap.com) (or equivalent) usage to make easier the graphical interfaces integration on the web. In addition, a large number of tools and frameworks integrate it natively.
+
+However, it is difficult to use Bootstrap in an Atomic Design approach. We quickly lose the consistency of our application in favor of Bootstrap's graphical identity.
+
+I want to share with you my experience of _Bootstrap_ so that you know how to make it cohabit with your own styles rather than fighting against it.
+
+# But, what is Bootstrap ?
+
+> What is Bootstrap ?
+
+This question may seem trivial at first sight, but it is not!
+
+From the official website, it is a framework for building [responsive](https://alistapart.com/article/responsive-web-design), [mobile-first](https://www.lukew.com/ff/entry.asp?933) web applications. From their remarks, it's the world's most popular front-end component library on the web and it's based on:
+
+* HTML structure
+* CSS style file (also Sass sources)
+* JavaScript for component behaviour (display a modal, alert messages, …) with libraries like jQuery.
+
+To help us, _Bootstrap_ give us a grid system to organise components using columns that will be adapted to the screen size (we will talk about it later in the blog post) and a lot of components ready to use.
```

Comment:
```suggestion
To help us, _Bootstrap_ gives us a grid system to organize components using columns that will be adapted to the screen size (we will talk about it later in the blog post) and a lot of components are ready to use.
```

Reply:
I used organise on my previous article, I know it's used in the UK, so, do you think I should use organize or organise for all this blog post?